Modules/modExtendSaveAsText.ACM

```diff
@@ -36,21 +36,9 @@ Private Const clngNOTFOUND = -2147483648#
 
 Private Declare Sub CopyMemory Lib "kernel32" Alias "RtlMoveMemory" (ByVal Destination As Any, ByVal Source As Any, ByVal Length As Long)
 
-'************************************************
-' This module extends out the saveAsText, loadFromText functionality of Access
-' to also be able to save tables, references, extras as well
-' This allows an entire Access Database to be exported into an SVN repository.
-'
-' @author Barry Pitfield (barry.pitfield@talkjcs.com)
-' @version 1.0
-'************************************************
-
-Public Sub SaveAsText(objType As asObjectType, objName As String, strFilename As String)
-  Dim Ignores(7) As Ignore
-  Dim blnSaveAsUTF8 As Boolean
-  Dim blnSourceIsUCS2 As Boolean
-  Dim blnFail As Boolean
-
+Private Sub PopIgnoreArr(ByRef Ignores() As IGNORE)

```

Comment:
nit: Rename to `PopulateIgnoreArr`? I thought this popped the first element off based on the name

Reply:
Done